### PR TITLE
Update Functions

### LINKED ISSUE #28: [FEATURE] Function names
* **What are you trying to do?**

The function names have become a bit messy. We currently have:

 * `audit_deps()` in v0.0.3
 * `audit_pkgs()`,  `audit_renv_lock()`,  `audit_req_txt()` in the dev branch.

The issues are:

 -  `audit_renv_lock()` and `audit_req_txt()` do exactly what you would expect
 - `audit_pkgs()` expects a vector of packages
 - `audit_deps()` is really odd, as the `deps` are either all installed packages, or a tibble of passed packages

* **What feature or behaviour is this required for?**

It's confusing (well it confuses me)

* **How could we solve this issue? (Not knowing is okay!)**

Rename the functions! Now is the time to break/deprecate things ;) Here are some quick suggestions, please suggest improvements

  - `audit_renv_lock()` and `audit_req_txt()` - look good
 - `audit_proj()` - this would scan an RStudio project for dependencies. For example, if an R package, it would read the DESCRIPTION file, and/or an env file
 - `audit_installed_pkgs()` - formally `audit_deps()`
 - `audit_pkgs()`  - don't really like this name, but can't think of something better. Perhaps just a general `audit()`?

* **Anything else?**

cc @bhamail / @DarthHater / @brittanybelle / @adrianpowell / @csgillespie / @JosiahParry

### PR DESCRIPTION
**Pull request by csgillespie**
* Fixes #28 
 * Breaking changes, so will bump version to 0.1.0 before CRAN
 * Standardise audit function names
 * Add codecoverage badge for testing

cc @bhamail / @DarthHater / @brittanybelle / @adrianpowell / @csgillespie / @JosiahParry

### CODE REVIEW

File: R/api_call.R [hunk 1]
```diff
@@ -61,7 +61,7 @@ get_post_authenticate = function(verbose) {
 }
 
 no_purls_case = function(verbose) {
-  results = tibble::tibble(package = character(0), description = character(0),
+  results = tibble::tibble(oss_package = character(0), description = character(0),
```
Comment (csgillespie) [inline R/api_call.R:64]: This was a bug

File: .travis.yml [hunk 2]
```diff
@@ -5,3 +5,8 @@ r:
   - oldrel
   - release
   - devel
+
+after_success: >
```
Comment (csgillespie) [inline .travis.yml:9]: Generate code coverage after travis

File: R/deprecated.R [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#'
+#' This function is deprecated. See
+#' @details By default, packages listed in \code{installed.packages()} are scanned by sonatype.
+#' However, you can pass your own data frame of packages. This data frame should have two columns,
+#' \code{version} and \code{package}.
+#' @param pkgs Default \code{NULL}. See details for further information.
+#' @param verbose Default \code{TRUE}.
+#' @return A tibble/data.frame.
+#' @export
+audit_deps = function(pkgs = NULL, verbose = TRUE) {
```
Comment (csgillespie) [inline R/deprecated.R:14]: Depreciate the previous function. Remove in future

Comment (JosiahParry) [reply R/deprecated.R:14]: Love it.

**Pull request merged**